specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/Jobs.json

```diff
@@ -625,6 +625,130 @@
+      "post": {
+        "tags": [
+          "Jobs"
+        ],
+        "summary": "Resumes a suspended job",
+        "operationId": "Jobs_Resume",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "jobName",
+            "in": "path",
+            "description": "Name of the Job.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Job resumed successfully.",
+            "schema": {
+              "$ref": "#/definitions/Job"
+            }
+          },
+          "202": {
+            "description": "Resume operation is in progress.",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string"
+              },
+              "Location": {
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Resume Job": {
+            "$ref": "./examples/Jobs_Resume.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/jobs/{jobName}/suspend": {
+      "post": {
+        "tags": [
+          "Jobs"
+        ],
+        "summary": "Suspends a job",
+        "operationId": "Jobs_Suspend",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "jobName",
+            "in": "path",
+            "description": "Name of the Job.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
```

Comment:
What does this regex do? I don't see it used for other jobName parameters

Reply:
under the Parameter section, i see the same regex is used for jobName? this is at line 761 in the new edited file

Reply:
oh...i see other apis just reference the Parameter...is that preferable?

Reply:
Right I think the same parameter should be referenced

Reply:
got it got it